spec/models/bishop_spec.rb

```diff
@@ -1,4 +1,21 @@
 require 'rails_helper'
 
 RSpec.describe Bishop, type: :model do
```

Comment:
Looking very good. So far I think we can get this merged into the master.

Reply:
Only suggestion, I would to be able to test valid move with the obstructed method. Maybe  create a separate RSPEC text, that makes a game, populates the game, then it moves the Pawn that is diagonal from a Bishop to be moved? Maybe move the pawn up one, and thus the test should still test true as working as the bishop is not blocked in that direction? But doing the other diagonal direction the test would false, as the pawn is there?

Reply:
I don't mind rewriting the code that way, but I'm also thinking that the is_obstructed? tests should be separate and testing specifically that method. @R-Currie @ivanperez002 @brihong - thoughts?

Reply:
Great suggestions to both @brephelps @ivanperez002 ! I would agree with Bre's approach to test the is_obstructed? separately in chess_piece.spec.rb 
I've shared my struggles with trying to pass the obstructions test in the rook_spec.rb heh 
However, Ivan if you were able to write a test for the Queen's obstruction, please share